torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -1246,5 +1244,54 @@ c10::optional<const Node*> AliasDb::getLastWildcard() const {
     return c10::nullopt;
   }
 }
+
+TORCH_API bool aliasAnalysisHasSpecialCaseFor(Symbol symbol) {
+  // WARNING: by adding a case to this list, you are asserting that you have
+  // added a case for the unschematized node in AliasDb::analyze
+  const static std::unordered_set<Symbol> handled = {
+    prim::If,
+    prim::Loop,
+    prim::FusionGroup,
+    prim::DifferentiableGraph,
+    prim::Constant,
+    prim::ListConstruct,
+    prim::TupleConstruct,
+    prim::Undefined,
+    prim::FusedConcat,
+    prim::MMTreeReduce,
+    prim::MMBatchSide,
+    prim::None,
+    prim::BroadcastSizes,
+    prim::ChunkSizes,
+    prim::Function,
+    prim::TupleUnpack,
+    prim::TupleIndex,
+    prim::TupleSlice,
+    prim::ListUnpack,
+    prim::PythonOp,
+    prim::ConstantChunk,
+    prim::BroadcastingChunk,
+    aten::add,
+    aten::sub,
+    aten::mul,
+    aten::div,
+  };
+
+  // Operators that should not be used by alias analysis
+  const static std::unordered_set<Symbol> purposefully_not_handled = {
+    prim::Print,
+    prim::Load,
+    prim::Store,
+    prim::Drop,
+    at::onnx::Reshape,
+    at::onnx::Shape,
+    prim::AnyDefined,
+    prim::AutogradAdd,
+    prim::fork,
```

Comment:
I don't think fork/wait is properly being handled currently.

if i do something like 

```
def identify(x):
    return x 
fut = torch.jit._fork(identify, x)
y = torch.jit._wait(fut)

```
From looking at the implementation it doesn't look like x & y will be in the same alias set